app/config/sources.inc.php

```diff
@@ -655,6 +655,10 @@
         'deadline'          => '2017-02-24',
         'supported_locales' => ['id', 'zh-TW'],
     ],
+    'snippets/2017/apr2017.lang' => [
+        'deadline'          => '2017-03-21',
+        'supported_locales' => ['de', 'es-ES', 'fr', 'id', 'pt-BR', 'ru', 'zh-TW'],
```

Comment:
I've just realized that we did 'es-ES' instead of 'es' here, not sure it made sense (the next month we went back to es).